odoo/addons/base/models/ir_mail_server.py

```diff
@@ -275,17 +296,28 @@ def test_smtp_connection(self):
                     # ignored, just a consequence of the previous exception
                     pass
 
-        message = _("Connection Test Successful!")
+        messages = [_('Connection Test Successful!')]
+        if autodetect_max_email_size:
+            if all(server.max_email_size for server in self):
+                messages.append(_('Email maximum size updated to %(size)s MB',
+                                  size=', '.join(f"{server.max_email_size:.2f}" for server in self)))
```

Comment:
As we're making a list of messages anyway, can't we have one message per server? "[Server name] email maximum size was updated to X MB" ?

Otherwise it's impossible to know which server was updated to what.

But honestly it's becoming very complex, as the form view will be updated with the new value now, do we really need to send that notification? We can simplify and directly raise if not supported above, the same way it's done for other issues.